dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_contentlets.jsp

```diff
@@ -806,7 +806,7 @@
         <!-- END Right Column -->
 
         <!-- START Show Query -->
-        <div id="queryDiv" dojoType="dijit.Dialog" title="<%= LanguageUtil.get(pageContext, "Query) %></div>" class="content-search__show-query-dialog" style="display: none;padding-top:15px\9;">
+        <div id="queryDiv" dojoType="dijit.Dialog" title="<%= LanguageUtil.get(pageContext, "Query") %></div>" class="content-search__show-query-dialog" style="display: none;padding-top:15px\9;">

```

Comment:
### Note

I only closed the `String literal` here. That was breaking the UI.


| Error   |  UI Effects | 
|---|---|
|  <img width="1416" alt="Screenshot 2023-01-26 at 11 25 41 AM" src="https://user-images.githubusercontent.com/72418962/214876104-d314fd26-a454-454c-8f3c-bb300b42d899.png">  |  <img width="873" alt="Screenshot 2023-01-26 at 11 26 08 AM" src="https://user-images.githubusercontent.com/72418962/214876208-3dd990d0-9daf-4c14-9f02-e26dc97ee96c.png">  |